File: pertemuan 5/Tugas5.py
nama = input("Masukan Nama Anda =")
fakultas = input("Masukan Fakultas Anda=")
prodi = input("Masukan prodi Anda =")
print("Nama saya Adalah =" + str(nama)) #Nama mahasiswa yang bersangkutan
print("fakultas =" + str(fakultas))
print("prodi =" + str(prodi))

ipk = input("Masukan IPK Anda =")
tahun = input("Masukan lama studi anda=")
ipk = float(ipk)
tahun = int(tahun)
if (ipk >= 3.51 and  ipk <= 4.00 and tahun <= 4):
    print("Anda lulus dengan pujian tertinggi atau prdikat summa Cumlaude")
elif (ipk >= 3.51 and ipk <= 4.00 and tahun > 4):
    print("Anda lulus dengan pujian atau predikat Cumlaude")
elif (ipk >= 3.01 and ipk < 3.51):
    print("Anda lulus dengan sangat memuaskan")
elif (ipk >= 2.76 and ipk <3.01):
    print("Anda lulus dengan memuaskan")
elif (ipk >= 2.00 and ipk < 2.76):
    print("Anda lulus dengan cukup")
elif (ipk > 0 and ipk < 2.00):
    print("Mohon maaf anda tidak lulus")
elif ( ipk> 4.00 or ipk  <0):
    print("Tidak valid")
else:
    print("Anda tidak terdeteksi")
print("Terima kasih")

File: pertemuan 5/Tugas5.java
import java.util.Scanner;

public class Tugas5 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Masukan nama Anda =");
        String nama = scanner.nextLine();

        System.out.print("Masukan Fakultas Anda =");
        String fakultas = scanner.nextLine();

        System.out.print("Masukan prodi anda =");
        String prodi = scanner.nextLine();
        
        System.out.println("Nama saya Adalah" + nama);
        System.out.println("fakultas =" + fakultas);
        System.out.println("prodi =" + prodi);

        System.out.print("Masukan Ipk Anda =");
        double ipk = scanner.nextDouble();
        System.out.print("Masukan lama studi belajar");
        int tahun = scanner.nextInt();
        if ( ipk >= 3.51 &&  ipk <= 4.00 && tahun <= 4) {
            System.out.println("Anda lulus dengan pujian tertinggi atau prdikat summa Cumlaude");
        } else if (ipk >= 3.51 && ipk <= 4.00 && tahun > 4) {
            System.out.println("Anda lulus dengan pujian atau predikat Cumlaude");
        } else if (ipk >= 3.01 && ipk < 3.51) {
            System.out.println("Anda lulus dengan sangat memuaskan");
        } else if (ipk >= 2.76 && ipk <3.01) {
            System.out.println("Anda lulus dengan memuaskan");
        } else if ( ipk >= 2.00 && ipk < 2.76) {
            System.out.println("Anda lulus dengan cukup");
        } else if (ipk > 0 && ipk < 2.00) {
            System.out.println("Mohon maaf anda tidak lulus");
        } else if (ipk> 4.00 || ipk  <0  ) {
            System.out.println("Tidak valid");
        } else {
            System.out.println("anda tidak terdeteksi");
        }
        System.out.println("Terima kasih");
    }
}
